### PR TITLE
APIGOV-18093 - note about catalog subscription mode

### PR DESCRIPTION
**Pull request by dfeldick**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by kflan-io — APPROVED** [review]

**Comment by cbordier** [comment]
@dfeldick, I am assuming this is a common behavior. Shouldn't we add the same note for other Gateways?

**Comment by dfeldick** [comment]
Yes, I guess so.

**Review by jcollins-axway — APPROVED** [review]

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md [hunk 1]
```diff
@@ -34,7 +34,8 @@ CENTRAL_SUBSCRIPTIONS_APPROVAL_MODE={manual|auto|webhook}
 CENTRAL_SUBSCRIPTIONS_APPROVAL_WEBHOOK_URL={The webhook URL that subscription data will be posted to}
 CENTRAL_SUBSCRIPTIONS_APPROVAL_WEBHOOK_HEADERS={The headers that will be used when posting data to the webhook url}
 ```
-
+{{< alert title="Note" color="primary" >}}Each API in Axway API Manager that is discovered will have the subscription approval mode for its corresponding catalog item in Amplify Central set to whatever the configured mode is at the time the API is discovered and published. Changing the configured mode and restarting the Discovery agent will not change the mode of any existing Amplify Central catalog items. To get the newly configured mode to be changed in any catalog items, the APIs in Axway API Manager must be rediscovered.{{< /alert >}}
```
[the remark this answers is not in the document]
Comment (dfeldick) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:37]: hmm. OK. Will fix.

[the remark this answers is not in the document]
Comment (dfeldick) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:37]: Odd. In netlify or VS code, I don't see this char. I even copy/paste this paragraph into the other 2 files and the char doesn't show up there.

**Comment by sbolosan** [comment]
Humor me, what is "Axway API Manager"?

**Comment by dfeldick** [comment]
Guess that's maybe wrong. In fact I guess I shouldn't have copy/pasted this paragraph. I need to do some fixing.

**Comment by sbolosan** [comment]
These updates are for AWS and Azure, but in cases where (at least) I used APIM, it usually means v7 only in reference to API Manager.  But I could be wrong myself...

**Comment by dfeldick** [comment]
I think you're right. My changes in AWS and Azure need to be changed to not say API Manager I guess.

**Review by sbolosan — APPROVED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes

File: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lbadenhop) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:37]:
```suggestion

{{< alert title="Note" color="primary" >}}Each API that is discovered in Axway API Manager contains the subscription approval mode for its corresponding Amplify Central catalog item, which is set when the API is discovered and published. The only way to change a catalog item's subscription approval mode is to rediscover the API in Axway API Manager.{{< /alert >}}
```

[the remark this answers is not in the document]
Comment (lbadenhop) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:37]: Please update all three notes. Cyrille has already approved the wording.

[the remark this answers is not in the document]
Comment (lbadenhop) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:37]: Use this wording but tweak for AWS API Gateway and Azure API Gateway:
Each API that is discovered in Axway API Manager contains the subscription approval mode for its corresponding Amplify Central catalog item, which is set when the API is discovered and published. The only way to change a catalog item's subscription approval mode is to update the agent configuration mode (refer to `CENTRAL_SUBSCRIPTIONS_APPROVAL_MODE`), restart the agent, and then rediscover the API in Axway API Manager.

[the remark this answers is not in the document]
Comment (dfeldick) [reply content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md:37]: looks good

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**